cmd/meroxa/root/connectors/remove.go

```diff
@@ -58,12 +58,12 @@ func (r *Remove) ValueToConfirm(_ context.Context) (wantInput string) {
 func (r *Remove) Execute(ctx context.Context) error {
 	r.logger.Infof(ctx, "Removing connector %q...", r.args.Name)
 
-	con, err := r.client.GetConnectorByName(ctx, r.args.Name)
+	con, err := r.client.GetConnectorByNameOrID(ctx, r.args.Name)
```

Comment:
we no longer need to make this API request if we can delete passing the name.

Reply:
we return the whole connector on line 73, that's the only reason that i kept it. it doesn't appear to be documented behavior in the official developer docs.  do you think anyone is relying on that object to be returned? any warnings before we change that behavior?

Reply:
Actually, never mind. Since `DeleteConnector` doesn't return the connector, we'll still need to fetch it for the json format.